src/nci_util.c

```diff
@@ -59,6 +60,10 @@ nci_listen_mode(
     return FALSE;
 }
 
+/*==========================================================================*
+ * Implementation
+ *==========================================================================*/
+
```

Comment:
That doesn't add much to readability, I would drop this comment. This file is just a bunch of assorted utilities, they are hard to combine into meaningful groups. These kinds of comments make more sense when you're implementing an object where you clearly have public interface, internal interface, methods, implementation and some internal stuff. But not in this case, IMO.